logical-comaprison-operators.js

```diff
@@ -6,35 +6,35 @@ TASK 1:
  // Have fun! // 😃
 ********************************************************************************/
 
-const exp1 = 10 >= 10; // TODO: ADD YOUR EVALUATION HERE -->
+const exp1 = 10 >= 10; // TODO: ADD YOUR EVALUATION HERE -- true>
 
-const exp2 = "dog" == "dog"; // TODO: ADD YOUR EVALUATION HERE -->
+const exp2 = "dog" == "dog"; // TODO: ADD YOUR EVALUATION HERE --true>
 
-const exp3 = true != false; // TODO: ADD YOUR EVALUATION HERE -->
+const exp3 = true != false; // TODO: ADD YOUR EVALUATION HERE --true>
 
-const exp4 = "10" === 10; // TODO: ADD YOUR EVALUATION HERE -->
+const exp4 = "10" === 10; // TODO: ADD YOUR EVALUATION HERE --fales>
 
-const exp5 = 5 > 4; // TODO: ADD YOUR EVALUATION HERE -->
+const exp5 = 5 > 4; // TODO: ADD YOUR EVALUATION HERE --true>
 
-const exp6 = null == undefined; // TODO: ADD YOUR EVALUATION HERE -->
+const exp6 = null == undefined; // TODO: ADD YOUR EVALUATION HERE --fales>
```

Comment:
This should return `true`, because both of these values are falsey values by default, meaning that `false == false`.

This would return `false` only when using the strict equals operator `===` which would also check for the types resulting in them being false.